src/main/java/org/jfrog/bamboo/builder/BuildInfoHelper.java

```diff
@@ -196,10 +200,9 @@ private void addBuildParentProperties(Map<String, String> propertyMap, TriggerRe
         }
     }
 
-    private static Module createModule(TaskContext taskContext, List<Artifact> artifacts, List<Dependency> dependencies, Map<String, String> properties) throws IOException {
+    private static Module createModule(List<Artifact> artifacts, List<Dependency> dependencies, String buildName, String buildNumber) {
```

Comment:
It makes sense for buildName and buildNumber to be the first args for this method.
The same goes for other methods in this class as well.

Reply:
Fixed.